README.md

```diff
@@ -162,15 +161,15 @@ without `--include-version-classifiers` only the major version will be included:
 -    License :: OSI Approved :: MIT License
 +    License :: OSI Approved :: MIT License
 +    Programming Language :: Python :: 3
-     Programming Language :: Python :: 3.6
+     Programming Language :: Python :: 3.8
 ```
 
 ### removes empty options in any section
 
 ```diff
  [options]
 -dependency_links =
- python_requires = >= 3.6.1
+ python_requires = >= 3.8
```

Comment:
I don't really plan to update these -- they're sorta intentionally out of date

Reply:
Makes it look like it's languishing a bit? But okay.